packages/jellyfish-api-core/__tests__/category/wallet/listWallets.test.ts

```diff
@@ -0,0 +1,38 @@
+import { Testing } from '@defichain/jellyfish-testing'
+import { MasterNodeRegTestContainer, RegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Wallet without masternode', () => {
+  const container = new RegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await client.wallet.createWallet('test')
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should listWallets', async () => {
+    await expect(client.wallet.listWallets()).resolves.toEqual(['', 'test'])
```

Comment:
```suggestion
    const wallets = await client.wallet.listWallets()
    expect(wallets).toStrictEqual(['', 'test'])
```

For test need to optimize for better readability, the above has 3 levels of complexity in one line, you should separate into 2 lines: first line to get the data, send to test the data.

Reply:
Noted, will generate more separated assignments from now on.